lib/platform/github/index.ts

```diff
@@ -384,7 +384,7 @@ export async function initRepo({
           }
         );
       } catch (err) /* istanbul ignore next */ {
-        logger.error(
+        logger.warn(
```

Comment:
I found this happened ephemerally and recovered next run, so should be OK to warn and not error